src/FixedDataTable.js

```diff
@@ -679,6 +690,8 @@ var FixedDataTable = createReactClass({
           cx('fixedDataTableLayout/main'),
           cx('public/fixedDataTable/main'),
         )}
+        tabIndex={0}
```

Comment:
It needed that to allow the component to get keyboard focus.  Scrollbar.js does the same thing.